### PR TITLE
Increase socket timeout while running migrations.

### PR DESCRIPTION
**Pull request by frankban**

### CODE REVIEW

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/charmstore/1891/
Test PASSed.

**Comment by urosj** [comment]
LGTM

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/charmstore/1892/
Test PASSed.

File: internal/charmstore/migrations.go [hunk 1]
```diff
@@ -90,6 +92,10 @@ type migration struct {
 
 // Migrate starts the migration process using the given database.
 func migrate(db StoreDatabase) error {
+	db = db.copy()
+	defer db.Close()
+	db.Session.SetSocketTimeout(10 * time.Minute)
+	defer db.Session.SetSocketTimeout(1 * time.Minute)
```
Comment (bac) [inline internal/charmstore/migrations.go:99]: Set it to 1 minute or to what it was prior?

**Comment by bac** [comment]
:+1:

File: internal/charmstore/migrations.go (hunk 1 above)
Comment (frankban) [reply internal/charmstore/migrations.go:99]: Done.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/charmstore/1893/
Test PASSed.

**Comment by frankban** [comment]
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/charmstore-merge

**Pull request merged**